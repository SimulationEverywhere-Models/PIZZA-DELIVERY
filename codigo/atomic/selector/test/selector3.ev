00:00:01:000 Carga_in 1
00:00:02:000 Carga_in 2
00:00:03:000 Esperar_m1 1
00:00:04:000 Carga_in 3
00:00:05:000 Carga_in 4
00:00:06:000 Esperar_m2 1
00:00:07:000 Esperar_m3 1
00:00:08:000 Carga_in 5
00:00:09:000 Carga_in 6
00:00:10:000 Esperar_m4 1
00:00:14:000 Lista_m3 1
00:00:15:000 Carga_in 7
00:00:16:000 Carga_in 8
00:00:17:000 Carga_in 9
00:00:18:000 Esperar_m3 1
00:00:19:000 Lista_m3 1
00:00:20:000 Carga_in 10