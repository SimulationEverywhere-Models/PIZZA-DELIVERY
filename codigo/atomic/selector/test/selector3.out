00:00:01:000 sel_out1 1
00:00:02:000 sel_out1 2
00:00:04:000 sel_out2 3
00:00:05:000 sel_out2 4
00:00:08:000 sel_out4 5
00:00:09:000 sel_out4 6
00:00:10:000 moto_ocupada 1
00:00:14:000 moto_libre 1
00:00:15:000 sel_out3 7
00:00:16:000 sel_out3 8
00:00:17:000 sel_out3 9
00:00:18:000 moto_ocupada 1
00:00:19:000 moto_libre 1
00:00:20:000 sel_out3 10
